### PR TITLE
Accounts

### PR DESCRIPTION
**Pull request by dougjones866**
Added accounts link to index.html which leads to accounts.html for sign up page. Also added to main.css.

From the user accounts issue https://github.com/lukeware/WATS-4020-Team-2/issues/5

### CODE REVIEW

**Comment by Okideveloper** [comment]
looks nice

File: contact.html [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+
+
+        <div class="container1">
+            <div style="text-align:center">
+                <h2>Contact Us</h2>
+                <p>If you have any questions or concerns please feel free to contact us.</p>
+            </div>
+            <div class="row">
+                <div class="column">
+                    <img src="/w3images/map.jpg" style="width:100%">
```
Comment (JohnMeyer32) [inline contact.html:108]: This image is broken and seems to throw off the formatting. We should change it.

**Comment by JohnMeyer32** [comment]
Looks great Doug. I found a broken image on the contacts page that we will need to fix. Also the formatting on the accounts page is set to a half page somehow which makes it difficult to use on mobile devices. Once those are done we can merge.

File: accounts.html [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+                    </li>
+                </ul>
+            </div>
+        </div>
+    </nav>
+
+
+
+    <!-- Page Content -->
+    <form action="action_page.php">
```
Comment (Okideveloper) [inline accounts.html:102]: With this kind of website, I do not think a registration is needed. Users visit this site to read poems and with the link I provided in the View public website branch, they will have an option to become a member to a community group. The contact page you created is all we need, I also made suggestions to that as well. Good work!

**Pull request closed without merging**